components/wifi_provisioning/src/manager.c

```diff
@@ -25,6 +25,16 @@
 
 #include "wifi_provisioning_priv.h"
 
+/**
+ * @brief Enables the SpinDance changes in wifi_prov_mgr_start_provisioning().
+ *
+ * The reasoning behind the SpinDance changes has been lost. Note that the code that remains
+ * (getting the current WiFi configuration) is effectively useless, because the restoring of
+ * the old WiFi config is performed inside the err: label but only if the WIFI_PROV_SETTING_BIT
+ * is set, and the code that sets the bit is commented out.
+ */
+#define INCLUDE_SPINDANCE_PROVISIONING_SETUP_CHANGES 1
```

Comment:
I was going to suggest we make this a KConfig value instead of a pre-processor macro, but since the reasoning for the changes this macro is wrapped around are lost, we couldn't possibly give the configuration a description that makes sense.